index.html

```diff
@@ -1,18 +1,49 @@
-
+<!DOCTYPE html>
 <html>
     <head>
-       <h1>Manizha khorram </h1>
+       <title>Manizha khorram </title>
+       <meta name="Author" content="Manizha khorram">
+       <link rel="stylesheet" type="text/css" href="Style.css" />
     </head>
+    <header>
+        
+      
     
+     <nav>
+        <a href="#About/">About</a> |
+        <a href="#Experience/">Experience</a> |
+        <a href="#Connect/">Connect</a> |
+        </nav>
+
+        <h1 id="myName">Manizha khorram</h1>
+
+        
+    </header>
     <body>
-        <h2>About</h2>
-        <p>I am from Afghanistan and I love coding. I would like to work as professional sofware developer. I hold bachelor of computer science degree, I love problem solving and creating new things.I keen to learn challenging roles in javaScript, software development and related technologies.</p>
-        <h2>Experience</h2>
-        <p>I have worked as a system development analyst at NPA (National Procurement Authority) and USWDP (University Work Force Development Program) in Afghanistan.</p>
-        <h2>Connect</h2>
-        <a  href="mailto:manizh.khorram@gmail.com">Gmail</a> <br>
-        <a  href="https://github.com/Manizha-khorram">GitHub account</a> <br>
-        <a  href="https://www.linkedin.com/in/manizha-khorram/">LinkedIn account</a> 
+        <img src="Developer Girl.jpeg" alt="Picture of a girl which is coding" id="position">
+        <section>
+            <a href="#About"><h1>About</h1> </a>
+            <p>I am from Afghanistan and I love coding. I would like to work as professional sofware developer. I hold bachelor of computer science degree, I love problem solving and creating new things.I keen to learn challenging roles in javaScript, software development and related technologies.</p>
+        </section>
+
+        <section>
+            <a href="#Experience"><h1>Experience</h1></a>
+            <p>I have worked as a system development analyst at NPA (National Procurement Authority) and USWDP (University Work Force Development Program) in Afghanistan.</p>
+        </section>
+
+        <section>
+            <a href="#Connect"> <h2>Connect</h2> </a>
+            <ul>
+              <li>  <a  href="mailto:manizh.khorram@gmail.com">Gmail</a>  </li><br>
+              <li> <a  href="https://github.com/Manizha-khorram">GitHub account</a>  </li> <br>
```

Comment:
I don't think the `<br>` tag is necessary at the end of this list item. There also seems to be some extra white space and some formatting inconsistencies here and there. :smile: :sweat_smile: